index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <title>mobile-view</title>
+    
+    <link rel="stylesheet" href="style.css">
+    
+</head>
+<body>
+    
+    <div class="wrapper">
+        <div class="wrapper1">
+          <header>
+            <div class="logo"><a href="index.html">My Logo</a></div>
+            <nav>
+              <a href="">
+                <img id="menu1" src="images/menu1.png" alt="menu" />
+              </a>
+            </nav>
+          </header>
```

Comment:
[ Suggestion Changes ]

- [ ] It would be nice you move the `header` tag to the line `13` so it can stand alone and place subsequent sections inside a `main` tag, which you can now give the wrapper class. This will make your code efficient and readable.